grid-cli/cmd/deploy_kubernetes.go

```diff
@@ -94,7 +94,7 @@ var deployKubernetesCmd = &cobra.Command{
 			return err
 		}
 
-		workersNode, err := cmd.Flags().GetUint32("workers-node")
```

Comment:
it's not a single node anymore, it's a slice of nodes so rename to workersNodes, workers-nodes